third_party/zlib/google/compression_utils_unittest.cc

```diff
@@ -107,7 +107,7 @@ TEST(CompressionUtilsTest, OutputCompressionAndDecompressionDuration) {
   std::string compressed_data;
```

Comment:
Please remove `#include "starboard/common/log.h"` at the top of the file as well, if possible.

Reply:
done